calci1.sh

```diff
@@ -13,7 +13,7 @@ echo "Enter your choice"
 read ch
 
 case $ch in
-1)sum=`expr $n1 - $n2`
+1)sum=`expr $n1 + $n2`
```

Comment:
can u plz add a comment.

Reply:
I hav implemented review comments,can you pls approve